unifi-controller/manifest.yaml

```diff
@@ -0,0 +1,6 @@
+name: Unifi-network-controller
```

Comment:
name should be all small and label you can add any 
Example 
```
name: falco-security
title: "Falco Security"
```

Reply:
@keithhubner can you make it like below 
```
name: unifi-network-controller
title: "Unifi network controller"
version: "stable-6"
```

Reply:
@keithhubner I cn merge once this is resolved

Reply:
Thanks, i've updated those bits, once it's in the marketplace I can do a full guide installing from the marketplace and also adding in an example of deploying with Traefik2 (hopefully also from the marketplace).